lib/Digitick/Sepa/DomBuilder/CustomerCreditTransferDomBuilder.php

```diff
@@ -80,20 +80,24 @@ public function visitPaymentInformation(PaymentInformation $paymentInformation)
             $this->createElement('CtrlSum', $this->intToCurrency($paymentInformation->getControlSumCents()))
         );
 
-        $paymentTypeInformation = $this->createElement('PmtTpInf');
-        if ($paymentInformation->getInstructionPriority()) {
-            $instructionPriority = $this->createElement('InstrPrty', $paymentInformation->getInstructionPriority());
-            $paymentTypeInformation->appendChild($instructionPriority);
-        }
-        $serviceLevel = $this->createElement('SvcLvl');
-        $serviceLevel->appendChild($this->createElement('Cd', 'SEPA'));
-        $paymentTypeInformation->appendChild($serviceLevel);
-        if ($paymentInformation->getCategoryPurposeCode()) {
-            $categoryPurpose = $this->createElement('CtgyPurp');
-            $categoryPurpose->appendChild($this->createElement('Cd', $paymentInformation->getCategoryPurposeCode()));
-            $paymentTypeInformation->appendChild($categoryPurpose);
+
+        // HCT domestic HUF payments are not SEPA, so we skip this
+        if ( $paymentInformation->getOriginAccountCurrency() !== 'HUF') {
```

Comment:
Here is one point we need to extract the building process for this xml part. I don't like to make it dependant on some currency here in the builder. This must be extracted to some factory or similar. I can't tell at the moment the best solution for this, but I have some vague ideas ;-)